### DISCUSSION TITLE
Is there an api for sending files to a specified device via command or interface

### DISCUSSION THREAD

**Discussion by bfrontend** [localsend/localsend#1051] [Discussion (English, International)]
### Is your feature request related to a problem? Please describe.

Is there an api for sending files to a specified device via command or interface
like support for  alfred or raycast

### Describe the solution you'd like

Is there an api for sending files to a specified device via command or interface
like support for  alfred or raycast

**Comment by Tienisto** [comment]
Yes, there is an API. Checkout https://github.com/localsend/protocol
